packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -237,12 +237,21 @@ function transformValue(
   value: number | OTDistribution | OTHistogram
 ) {
   if (isDistributionValue(value)) {
-    // TODO: Add support for Distribution
-    throw new Error('Distributions are not yet supported');
+    throw Error('unsupported distribution value type');
+    // no buckets aggregated, which is a required param in `distributionValue` for Cloud Monitoring v3
   }
   if (isHistogramValue(value)) {
-    // TODO: Add support for Histogram
-    throw new Error('Histograms are not yet supported');
+    return {
+      distributionValue: {
+        // sumOfSquaredDeviation param not aggregated
+        count: value.count,
+        mean: value.sum / value.count,
+        bucketOptions: {
+          explicitBuckets: { bounds: value.buckets.boundaries },
```

Comment:
Just want to double check that the JS SDK and Cloud Monitoring are using the same histogram boundary format. IIUC Cloud Monitoring uses a format where `[1, 5, 10]` would give intervals `(-inf, 1), [1, 5), [5, 10), [10, inf)` https://cloud.google.com/monitoring/api/ref_v3/rpc/google.api#explicit

Is OT JS using the same format?

Reply:
Hi @aabmass, sorry for the delay. From my testing, it seemed to me like they were using the same histogram boundary format, but I found that some inline comments which seemed to suggest the opposite when I double-checked after your comment. Those inline comments were incorrect and I made a PR to fix them / add a test: https://github.com/open-telemetry/opentelemetry-js/pull/1427

So in conclusion, the JS SDK and Cloud Monitoring are using the same histogram boundary format :)